julia/LibCEED.jl/src/generated/libceed_bindings.jl

```diff
@@ -1350,6 +1350,10 @@ function CeedHouseholderApplyQ(mat_A, mat_Q, tau, t_mode, m, n, k, row, col)
     ccall((:CeedHouseholderApplyQ, libceed), Cint, (Ptr{CeedScalar}, Ptr{CeedScalar}, Ptr{CeedScalar}, CeedTransposeMode, CeedInt, CeedInt, CeedInt, CeedInt, CeedInt), mat_A, mat_Q, tau, t_mode, m, n, k, row, col)
 end
 
+function CeedMatrixPseudoinverse(ceed, mat, m, n, mat_pinv)
```

Comment:
I've actually been trying to get rid of the other language bindings for utility functions like this - in Julia, Python, and Rust, you have better ways to do this

Reply:
Hmm, okay I'll remove it, how can I do that in a better way?

Reply:
In Julia, I'd use  backslash, for example, instead of anything libCEED offers